### ISSUE TITLE
Install failing, Can't locate object method "new" via package "JSON::Schema::Modern"

### ISSUE THREAD

**Issue by lschierer** [karenetheridge/Mojolicious-Plugin-OpenAPI-Modern#32]
when I run ```cpanm Mojolicious::Plugin::OpenAPI::Modern``` it fails its tests, and the log reports that it cannot locate object method "new" via package "JSON::Schema::Modern".  It continues to do this even though 
```
luke$ cpanm JSON::Schema::Modern
JSON::Schema::Modern is up to date. (0.593)
luke$
```
[build.log](https://github.com/user-attachments/files/17499998/build.log)

I'm attaching the build.log file.

**Comment by karenetheridge** [comment]
Neat, someone is using this!

Fixed in OpenAPI::Modern 0.072.

**Comment by lschierer** [comment]
confirmed fixed, I saw the update a little while ago. Thanks!!
